packages/client/components/AnalyticsPage.tsx

```diff
@@ -16,6 +17,7 @@ const query = graphql`
   query AnalyticsPageQuery {
     viewer {
       id
+      segmentId
```

Comment:
We also initialize segment on this page but never read the `segmentId` before, which strikes me as odd. Are you sure we're writing this before we query it?